### DISCUSSION TITLE
Unavailable light bulbs: Trådfri bulbs more often affected than Innr?

### DISCUSSION THREAD

**Discussion by m0wlheld** [Koenkk/zigbee2mqtt#5806] [Devices]
Dear all,

in an effort to make perimeter's lightning "smart", I started with etablishing a Zigbee network. For hardware, I use a Raspberry Pi 4B 4GB with the Raspbee II add-on card. Software are Zigbee2MQTT, Mosquitto and Home Assistant, all running in Docker containers on the latest, stable version (no -dev).

So far, I have installed and paired 26 light bulbs for E27 sockets of which 18 are [Ikea Trådfri LED1836G9](https://www.zigbee2mqtt.io/devices/LED1836G9.html), while the remaining 8 are [Innr RB 265](https://www.zigbee2mqtt.io/devices/RB_265.html).

Looking at Home Assistant's history representation, I see that many lamps go offline and online again. What's notable: The lamps using Trådfri bulbs '("Lampe Zufahrt","Lampe Fussweg") are more often / almost exclusively subject to missing connectivity. The Innr bulbs ("Lampe Haus 30", "Lampe Sitzplatz") are less affected, almost none. 

Also noteable: Although some bulbs are reported offline, they all react properly when addressed using Zigbee groups. 

What could be the reason for this behaviour, is it a timing issue (already increased `availability_timeout` to 90sec) or do i really have connectivity issues, that need to be solved.

Please see the following schematics:

- the coordinator is in the center of the red disk
- green ("Lampe Sitzplatz") and black dots framed red ("Lampe Haus 30") are Innr bulbs
- remaining black dots are Ikea bulbs
- blue dots are yet to be installed

![grafik](https://user-images.githubusercontent.com/23745709/104927616-f205b880-59a1-11eb-9f4f-ca06b886d854.png)

That's the output of the last 30mins for a single selected bulb:

```
info  2021-01-18 15:05:34: MQTT publish: topic 'zigbee2mqtt/0x------fffe52bf63/availability', payload 'offline'
error 2021-01-18 15:05:34: Failed to ping '0x------fffe52bf63'
info  2021-01-18 15:07:04: MQTT publish: topic 'zigbee2mqtt/0x------fffe52bf63/availability', payload 'online'
info  2021-01-18 15:07:05: MQTT publish: topic 'zigbee2mqtt/0x------fffe52bf63', payload '{"brightness":254,"linkquality":37,"state":"OFF","update":{"state":"idle"}}'
info  2021-01-18 15:07:05: MQTT publish: topic 'zigbee2mqtt/0x------fffe52bf63', payload '{"brightness":254,"linkquality":37,"state":"OFF","update":{"state":"idle"}}'
info  2021-01-18 15:17:07: MQTT publish: topic 'zigbee2mqtt/0x------fffe52bf63/availability', payload 'offline'
error 2021-01-18 15:17:07: Failed to ping '0x------fffe52bf63'
info  2021-01-18 15:18:38: MQTT publish: topic 'zigbee2mqtt/0x------fffe52bf63/availability', payload 'online'
error 2021-01-18 15:19:38: Failed to read state of '0x------fffe52bf63' after reconnect
info  2021-01-18 15:33:12: MQTT publish: topic 'zigbee2mqtt/0x------fffe52bf63/availability', payload 'offline'
error 2021-01-18 15:33:12: Failed to ping '0x------fffe52bf63'
```

Home Assistant history view. Red is online but switched off, gray is offline.

![grafik](https://user-images.githubusercontent.com/23745709/104926441-735c4b80-59a0-11eb-912a-0cbaea6bb491.png)

**Comment by miguelarios** [comment]
I too have experienced my light bulbs toggle from unavailable to available often a couple of times per hour. This happens for all bulbs, so distance from the controller doesn't seem to be the cause, and I have the zig-a-zig controller so it is a powerful one already.

I have, this morning, configured the `availability_timeout` down to 30s vs 60s and still noticing the bulbs toggling between unavailable and available. 

Unsure what else to do.
